atomic_reactor/schemas/workflow_data.json

```diff
@@ -0,0 +1,194 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "atomic-reactor container build workflow data",
+  "type": ["object", "null"],
+  "properties": {
+    "dockerfile_images": {"$ref": "#/definitions/dockerfile_images"},
+    "tag_conf": {"$ref": "#/definitions/tag_conf"},
+    "push_conf": {"$ref": "#/definitions/push_conf"},
+
+    "prebuild_results": {"type": "object"},
+    "buildstep_result": {"type": "object"},
+    "postbuild_results": {"type": "object"},
+    "prepub_results": {"type": "object"},
+    "exit_results": {"type": "object"},
+    "plugin_workspace": {"type": "object"},
+
+    "plugins_timestamps": {"type": "object"},
+    "plugins_durations": {"type": "object"},
+    "plugins_errors": {"type": "object"},
+    "build_canceled": {"type": "boolean"},
+    "plugin_failed": {"type": "boolean"},
+
+    "reserved_build_id": {"type": ["integer", "null"], "minimum": 1},
+    "reserved_token": {"type": ["string", "null"]},
+    "koji_source_nvr": {
+      "type": "object",
+      "properties": {
+        "name": {"type": "string"},
+        "version": {"type": "string"},
+        "release": {"type": "string"}
+      }
+    },
+    "koji_source_source_url": {"type": ["string", "null"]},
+    "koji_source_manifest": {"type": ["object", "null"]},
+
+    "buildargs": {"type": "object"},
+    "built_image_inspect": {"type": ["object", "null"]},
+    "layer_sizes": {
+      "type": "array",
+      "items": {"$ref": "#/definitions/layer_size"}
+    },
+    "pulled_base_images": {"type": "array", "uniqueItems": true},
```

Comment:
why `true` is red here?

Reply:
well we don't even need pulled_base_images :)